src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -0,0 +1,693 @@
+
+        public static Color RoyalBlue => new Color(KnownColor.RoyalBlue);
+
+        public static Color SaddleBrown => new Color(KnownColor.SaddleBrown);
+
+        public static Color Salmon => new Color(KnownColor.Salmon);
+
+        public static Color SandyBrown => new Color(KnownColor.SandyBrown);
+
+        public static Color SeaGreen => new Color(KnownColor.SeaGreen);
+
+        public static Color SeaShell => new Color(KnownColor.SeaShell);
+
+        public static Color Sienna => new Color(KnownColor.Sienna);
+
+        public static Color Silver => new Color(KnownColor.Silver);
+
+        public static Color SkyBlue => new Color(KnownColor.SkyBlue);
+
+        public static Color SlateBlue => new Color(KnownColor.SlateBlue);
+
+        public static Color SlateGray => new Color(KnownColor.SlateGray);
+
+        public static Color Snow => new Color(KnownColor.Snow);
+
+        public static Color SpringGreen => new Color(KnownColor.SpringGreen);
+
+        public static Color SteelBlue => new Color(KnownColor.SteelBlue);
+
+        public static Color Tan => new Color(KnownColor.Tan);
+
+        public static Color Teal => new Color(KnownColor.Teal);
+
+        public static Color Thistle => new Color(KnownColor.Thistle);
+
+        public static Color Tomato => new Color(KnownColor.Tomato);
+
+        public static Color Turquoise => new Color(KnownColor.Turquoise);
+
+        public static Color Violet => new Color(KnownColor.Violet);
+
+        public static Color Wheat => new Color(KnownColor.Wheat);
+
+        public static Color White => new Color(KnownColor.White);
+
+        public static Color WhiteSmoke => new Color(KnownColor.WhiteSmoke);
+
+        public static Color Yellow => new Color(KnownColor.Yellow);
+
+        public static Color YellowGreen => new Color(KnownColor.YellowGreen);
+
+        //
+        //  end "web" colors
+        // -------------------------------------------------------------------
+
+        // NOTE : The "zero" pattern (all members being 0) must represent
+        //      : "not set". This allows "Color c;" to be correct.
+
+        private static short s_stateKnownColorValid = 0x0001;
+        private static short s_stateARGBValueValid = 0x0002;
+        private static short s_stateValueMask = (short)(s_stateARGBValueValid);
+        private static short s_stateNameValid = 0x0008;
+        private static long s_notDefinedValue = 0;
+
+        /**
+         * Shift count and bit mask for A, R, G, B components in ARGB mode!
+         */
+        private const int ARGBAlphaShift = 24;
+        private const int ARGBRedShift = 16;
+        private const int ARGBGreenShift = 8;
+        private const int ARGBBlueShift = 0;
+
+
+        // user supplied name of color. Will not be filled in if
+        // we map to a "knowncolor"
+        //
+        private readonly string name;
+
+        // will contain standard 32bit sRGB (ARGB)
+        //
+        private readonly long value;
```

Comment:
Is there a particularly good reason for this to use a 64 bit long rather than a 32 bit type such as uint? It seems like it is using double the memory needed.